rlclientlib/azure_factories.cc

```diff
@@ -31,13 +31,34 @@ namespace reinforcement_learning {
   }
 
   int restapi_data_transport_create(m::i_data_transport** retval, const u::configuration& config, i_trace* trace_logger, api_status* status) {
-    const auto uri = config.get(name::MODEL_BLOB_URI, nullptr);
+    auto uri = config.get(name::MODEL_BLOB_URI, nullptr);
+    auto usingAPI = false;
```

Comment:
double check if this repo is using camel naming or not. looks no

Reply:
also the naming is unclear

Reply:
renamed it to using_api_endpoint